src/renderer/globals.scss

```diff
@@ -190,3 +234,33 @@ a.is-button {
 .modal-card {
   z-index: 200;
 }
+
+.button .is-rounded .is-primary {
+  color: $pitch !important;
```

Comment:
are we sure to use `dark` color for button?

Reply:
this is the color of the text in frequency background
<img width="146" alt="Untitled" src="https://github.com/rfcx/arbimon-uploader/assets/31901584/f7347f43-1840-4c59-b9f1-1344f1c9c117">